libskk/rom-kana.vala

```diff
@@ -254,7 +267,10 @@ namespace Skk {
                 if (index >= 0) {
                     index = PERIOD_RULE[period_style].index_of_nth_char (index);
                     unichar period = PERIOD_RULE[period_style].get_char (index);
-                    _output.append_unichar (period);
+                    string kana_period = Util.convert_by_input_mode(
+                        period.to_string(),
+                        kana_mode.to_input_mode());
```

Comment:
I guess you could simply cast it to ```InputMode``` with ```(InputMode)kana_mode```, as the values in these enums are compatible: https://github.com/ueno/libskk/blob/master/libskk/context.vala#L40